src/app/components/core/north/north-task-modal/north-task-modal.component.ts

```diff
@@ -234,17 +234,16 @@ export class NorthTaskModalComponent implements OnInit, OnChanges {
   }
 
   public saveScheduleFields(form: NgForm) {
-    if (!form.dirty && !form.touched) {
```

Comment:
the check looks valid. does deleting a filter / re-order does not mark form dirty and or touched?